wave_lang/kernel/ops/wave_ops.py

```diff
@@ -3024,6 +3029,14 @@ def infer_type(self, *args):
         self.type = get_custom(_to_sequence(self.args)[0]).type
 
 
+@define_op("tensor_load_to_lds")
+@dataclass
+class TensorLoadToLDS(CustomOp):
+    src: Memory
+    dst: Memory
+    descriptors: list = field(default_factory=list)
```

Comment:
We should explicitly encode tile sizes/strides/index like you did in `TensorLoadConfig` instead of abstract descriptor list.